src/db.py

```diff
@@ -0,0 +1,34 @@
+__author__ = 'Tsintsir'
+
+from binascii import b2a_hex
+from os import urandom
+from pymongo import MongoClient
+from pbkdf2 import PBKDF2
+
+
+def create_user(username, email, password):
```

Comment:
Not at all, that sounds reasonable enough! Not giving out information on login failure makes sense for security. I just wanted to make sure I understood correctly.